UPGRADE-9.0.md

```diff
@@ -0,0 +1,20 @@
+UPGRADE FROM 8.x to 9.0
+=======================
+
+* Interaction Builder
+  * It's now required to call `PhpPact\Consumer\InteractionBuilder::createMockServer` manually
+
+   Example Usage:
+   ```php
+   $builder = new InteractionBuilder($config);
+   $builder
+       ->given('a person exists')
+       ->uponReceiving('a get request to /hello/{name}')
+       ->with($request)
+       ->willRespondWith($response);
+   $builder->createMockServer();
+
+   $apiClient->sendRequest();
+
+   $this->assertTrue($builder->verify());
+   ```
```

Comment:
Do you have any better idea? e.g. Move `createMockServer` inside `willRespondWith` so we don't have to call `createMockServer` manually at all?

If so do we have plan to support multiple interactions per builder/mock server? e.g.

```php
$builder
    ->given()
    ...
    ->willResponseWith();
$builder
    ->newInteraction()
    ->given()
    ...
    ->willResponseWith();
$builder->createMockServer();
```

Reply:
After watching this video https://docs.pact.io/consumer#watch-a-video-writing-good-consumer-tests + practicing writing some http pacts, I think it's no need to introduce this breaking change.

So I remove this breaking change by doing:

> Move createMockServer inside willRespondWith so we don't have to call createMockServer manually at all

About this:

> Do we have plan to support multiple interactions per builder/mock server? 

I think (at least for me) we don't need this feature. If it's really needed we can work on it later.